portfolio/src/main/webapp/index.html

```diff
@@ -8,27 +8,63 @@
     </head>
     <body>
     <header>
-        denzilbilson
-        <nav id="menu">
+        <strong>&lt;denzilbilson /&gt;</strong>
             <ul>
                 <li><a href="#home">Home</a></li>
                 <li><a href="#about-me">About Me</a></li>
                 <li><a href="#projects">Projects</a></li>
                 <li><a href="#contact-me">Contact Me</a></li>
             </ul>
-        </nav>
     </header>
         <div id="content">
             <section id="home">
-                <h1>Welcome</h1> 
-                <h2>My name is Denzil Ekow Bilson, and I am a budding Software Engineer.</h2>
-                <p>My goal is to help create and foster diversity and inclusion throughout the tech field.</p>
+                <div id="text">
+                    <h2> Welcome. <br /> My name is Denzil Bilson, and I am a budding Software Engineer.</h2>
+                </div>
+                <img src="/images/denzil.jpg" id="denzilpicture" >
             </section>
             <section id="about-me">
+                <div id="experiences">
+                    <div class="box-block">
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/6/61/HTML5_logo_and_wordmark.svg">
+                        </div>
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/3/3d/CSS.3.svg">
+                        </div>
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/b/ba/Javascript_badge.svg">
+                        </div>
+                    </div>
+                    <div class="box-block">
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/1/18/ISO_C%2B%2B_Logo.svg">
+                        </div>
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/c/c3/Python-logo-notext.svg">
+                        </div>
+                        <div class="box">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg">
+                        </div>
+                    </div>
+                </div>
+                <div id="text">
```

Comment:
There should only be one element with a given ID on a page; maybe `text` should be a class, instead?